sirepo/package_data/static/js/sirepo.js

```diff
@@ -2147,4 +2225,47 @@ SIREPO.app.controller('SimulationsController', function (appState, panelState, r
         appState.saveQuietly('simulationFolder');
     });
     loadList();
+
+    // invoked in loadList() callback
+    function checkURLForFolder() {
+
+        var pathNoSectionRE = new RegExp('^\/' + activeSection.getActiveSection());
```

Comment:
This shouldn't be a regular expression. getActiveSection needs to be escaped if it is, but more importantly, it's literal match below.